README.md

```diff
@@ -128,78 +213,21 @@ The Azure Pipelines Infracost tasks can be used with either Azure Repos (only gi
-            displayName: Install Terraform
-
-          # IMPORTANT: add any required steps here to setup cloud credentials so Terraform can run
-
-          - bash: terraform init
-            displayName: Terraform init
-            workingDirectory: $(TF_ROOT)
-
-          - bash: terraform plan -out tfplan.binary
-            displayName: Terraform plan
-            workingDirectory: $(TF_ROOT)
-
-          - bash: terraform show -json tfplan.binary > plan.json
-            displayName: Terraform show
-            workingDirectory: $(TF_ROOT)
-
-          # Install the Infracost CLI, see https://github.com/infracost/infracost-azure-devops#infracostsetup
-          # for other inputs such as version, and pricingApiEndpoint (for self-hosted users).
-          - task: InfracostSetup@0
-            displayName: Setup Infracost
-            inputs:
-              apiKey: $(infracostApiKey)
-
-          # Run Infracost and generate the JSON output, the following docs might be useful:
-          # Multi-project/workspaces: https://www.infracost.io/docs/features/config_file
-          # Combine Infracost JSON files: https://www.infracost.io/docs/features/cli_commands/#combined-output-formats
-          - bash: infracost breakdown --path=$(TF_ROOT)/plan.json --format=json --out-file=/tmp/infracost.json
-            displayName: Run Infracost
-
-          # See https://github.com/infracost/infracost-azure-devops#infracostcomment for other options
-          - task: InfracostComment@0
-            displayName: Post Infracost comment
-            inputs:
-              githubToken: $(githubToken) # Required to post comments
-              path: /tmp/infracost.json
-              # Choose the commenting behavior, 'update' is a good default:
-              behavior: update # Create a single comment and update it. The "quietest" option.
-              # behavior: delete-and-new # Delete previous comments and create a new one.
-              # behavior: hide-and-new # Minimize previous comments and create a new one.
-              # behavior: new # Create a new cost estimate comment on every push.
-              # Limit the object that should be commented on, either merge-request or commit
-              # targetType: pull-request
-    ```
-
-4. Click 'Save and run'
-
-5. 🎉 That's it! Send a new pull request to change something in Terraform that costs money. You should see a pull request comment that gets updated, e.g. the 📉 and 📈 emojis will update as changes are pushed!
+#### 503 error when posting to Azure Devops Repo
+If you receive a 503 error when running the `InfracostComment` task in your pipeline:
+![](https://github.com/infracost/infracost-azure-devops/blob/master/.github/assets/503.png)
+This is normally because the build agent does not have permissions to post to the Azure Devops Repo. Make sure step 3 (Enable Azure Pipelines to post pull request comments) of the [#azure-repos-quick-start](Azure Repos Quick start) is complete. 
```

Comment:
```suggestion
#### 403 error when posting to Azure Repo
If you receive a 403 error when running the `InfracostComment` task in your pipeline:
![](https://github.com/infracost/infracost-azure-devops/blob/master/.github/assets/403.png)
This is normally because the build agent does not have permissions to post to the Azure Repo. Make sure step 3 (Enable Azure Pipelines to post pull request comments) of the [#azure-repos-quick-start](Azure Repos Quick start) is complete. 
```

Reply:
whoops!